language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/components/BotTest.scala

```diff
@@ -151,6 +151,16 @@ class BotTest extends FlatSpec with Matchers with DataLayerHelpers {
                   commands))
               Single.error(new RuntimeException("expected failure"))
             }
+            // TODO Untested endpoint
```

Comment:
Rather than a `TODO`, perhaps:

```scala
throw new UnsupportedOperationException("untested endpoint; implement when you need it")
```

Reply:
I'm trying to move the clients in parallel and this is a single file that everybody touches, I'll leave it alone for the time being but I will definitely apply this suggestion when I'm putting all the clients together in the final PR for this ticket. Thanks! :+1: